packages/dashboards/addon/templates/dashboards/dashboard/widgets/widget.hbs

```diff
@@ -119,9 +119,17 @@
   {{!-- Footer --}}
 
   <footer class="navi-report-widget__footer">
-    {{#if (not isRunningReport) class="navi-report-widget__footer-btns"}}
-
+    {{#if isRunningReport}}
+      <button
```

Comment:
Is this addition necessary for this PR? It might be best to add `cancelReport` to the widget view in a separate PR and add a test around it just so we know it works in that view as well as the regular report view.